tensorbay/apps/sextant.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Interact with sextant app at graviti marketplace."""
+
+from typing import Any, Dict, List, Optional
+from urllib.parse import urljoin
+
+from tensorbay.client.requests import Client
+
+
+class Evaluation:
+    """This class defines :class:`Evaluation`.
+
+    Arguments:
+        evaluation_id: Evaluation ID.
+        benchmark: The :class:`Benchmark`.
+
+    """
+
+    def __init__(self, evaluation_id: str, benchmark: "Benchmark") -> None:
+        self.evaluation_id = evaluation_id
+        self.benchmark = benchmark
+
+    def get_result(self) -> Dict[str, Any]:
+        """Get the result of the evaluation.
+
+        Return:
+            The result dict of the evaluation.
+
+        """
+
+    def get_status(self) -> str:
+        """Get the status of the evaluation.
+
+        Return:
+            One of the Success, failed or processing.
+
+        """
+
+
+class Benchmark:  # pylint: disable=too-many-instance-attributes
+    """This class defines :class:`Benchmark`.
+
+    Arguments:
+        name: Name of the Benchmark.
+        dataset_id: ID of the dataset which on this benchmark based.
+        commit_id: ID of the commit which is used as the evaluation benchmark.
+        benchmark_id: Benchmark ID.
+        sextant: The :class:`SextantClient`.
+        categories: The needed evaluation categories, if not given, all categories will be used.
+        iou_threshold: The IoU threshold.
+        customized_metrics: Https url of the github repository.
+
+    """
+
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        name: str,
+        dataset_id: str,
+        commit_id: str,
+        benchmark_id: str,
+        sextant: "Sextant",
+        categories: Optional[List[str]] = None,
+        iou_threshold: Optional[float] = None,
+        customized_metrics: Optional[str] = None,
```

Comment:
Should these three arguments be set as keyword only like in `Sextant.create_benchmark()`?